test/fixtures/core.scale/label-align-inner.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  config: {
+    type: 'line',
+    data: {
+      datasets: [{
+        data: [1, 2, 3],
+      }],
+      labels: ['Label1', 'Label2', 'Label3']
+    },
+    options: {
+      scales: {
+        x: {
+          ticks: {
+            align: 'inner',
+          },
+        },
+        y: {
+          ticks: {
+            align: 'inner',
+          }
+        }
```

Comment:
Would remove this, since it only applies to horizontal axes this will give confusion